components/timezone/src/provider.rs

```diff
@@ -70,6 +74,82 @@ impl<'a> zerovec::maps::ZeroMapKV<'a> for MetaZoneId {
     type OwnedType = MetaZoneId;
 }
 
+/// MinutesSinceEpoch is a wall-clock time represented as the number of minutes since the local unix epoch.
+#[repr(transparent)]
+#[derive(Debug, Clone, Copy, Eq, Ord, PartialEq, PartialOrd, yoke::Yokeable, ULE, Hash)]
+pub struct MinutesSinceEpoch(i32);
+
+impl AsULE for MinutesSinceEpoch {
+    type ULE = Self;
+
+    #[inline]
+    fn to_unaligned(self) -> Self::ULE {
+        self
+    }
+
+    #[inline]
+    fn from_unaligned(unaligned: Self::ULE) -> Self {
+        unaligned
+    }
+}
+
+impl<'a> zerovec::maps::ZeroMapKV<'a> for MinutesSinceEpoch {
+    type Container = ZeroVec<'a, MinutesSinceEpoch>;
+    type Slice = ZeroSlice<MinutesSinceEpoch>;
+    type GetType = MinutesSinceEpoch;
+    type OwnedType = MinutesSinceEpoch;
+}
+
+#[cfg(feature = "datagen")]
+impl serde::Serialize for MinutesSinceEpoch {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+        where
+            S: serde::Serializer,
+    {
+        let string = DateTime::from_minutes_since_local_unix_epoch(26382240)?.to_string();
+        serializer.serialize_str(&string)
+
+    }
+}
+
+pub(crate) struct DeserializeMinutesSinceEpochString;
+
+impl<'de> serde::de::Visitor<'de> for DeserializeMinutesSinceEpochString {
```

Comment:
there are some errors complaining like `failed to resolve: use of undeclared crate or module `de`` or cannot find `Deserialize`.

I tried to use sth like `use ::serde::{de, Deserialize, Deserializer};` from other examples, but had no luck.

Reply:
Add the annotation `#[cfg(feature = "serde")]`